src/Auth0.AuthenticationApi/Builders/SamlUrlBuilder.cs

```diff
@@ -1,64 +1,74 @@
+using System;
+using System.Collections.Generic;
 using System.Linq;
 
 namespace Auth0.AuthenticationApi.Builders
 {
     /// <summary>
-    /// Used to build a SAML authorization URL.
+    /// Builder class used to fluently construct a SAML authorization URL.
     /// </summary>
+    /// <remarks>
+    /// See https://auth0.com/docs/api/authentication#accept-request for more details.
+    /// </remarks>
     public class SamlUrlBuilder : UrlBuilderBase<SamlUrlBuilder>
     {
         /// <summary>
         /// Initializes a new instance of the <see cref="SamlUrlBuilder"/> class.
         /// </summary>
-        /// <param name="baseUrl">The base URL.</param>
-        /// <param name="client">The client id.</param>
-        public SamlUrlBuilder(string baseUrl, string client)
-            :base(baseUrl, "samlp/{client}")
+        /// <param name="baseUrl">Base URL of the Authentication API represented as a <see cref="String"/>.</param>
+        /// <param name="clientId">Client ID of the application.</param>
+        public SamlUrlBuilder(string baseUrl, string clientId)
+            : base(baseUrl, "samlp/{client}")
```

Comment:
won't it be more readable if this changes to
```suggestion
            : base(baseUrl, "samlp/{clientId}")
```

and below you keep "setting" the segment `clientId` as well?

(there are a few places in this diff that this comment applies)